### ISSUE TITLE
Output type "book" should put a page break before a new part

### ISSUE THREAD

**Issue by vogella** [asciidoctor/asciidoctor-pdf#473]
I believe the pdf output should add a page break before a new part if the book style is used. It does already add a page break after the part and I do not see how it can be useful to have the header of the part on the page before.

Currently, I have to put the page break in manually. Example:
````
[abstract]
This script give an introduction into basic Android development tasks.

<<<
= Introduction to Android development

:imagesdir: ../com.vogella.tutorials.android/Android/img

include::../com.vogella.tutorials.android/Android/010_overview.adoc[]
include::../com.vogella.tutorials.android/Android/140_manifestmf.adoc[]

<<<
= Android Studio installation

include::../com.vogella.tutorials.android/Android/016_androidstudioinstallation.adoc[]
include::../com.vogella.tutorials.android/Android/024_tutorial_firstproject_androidstudio.adoc[]
````

**Comment by mojavelinux** [comment]
This is a duplicate of #74.

**Comment by vogella** [comment]
Marking as closed.

**Comment by mojavelinux** [comment]
The more specific issue is #329, which is resolved for the upcoming 1.5.0.alpha.13 release.
